.gitignore

```diff
@@ -1,7 +1,7 @@
 ##Ignore site configuration
 .gitmodules
 site-config/*
-
+.idea
```

Comment:
Is that nessecary across sites or is that a developer specific (e.g. IDE) file?

Reply:
The development of the bridgehead doesn't necessarily require IntelliJ. However, the most used current IDEs for developing it are IntelliJ and Visual Studio. Even if it is not completely necessary, this short line can avoid many problems for the developers. I would suggest removing it when IntelliJ is no longer used in the department.